docs/index.md

```diff
@@ -16,6 +16,10 @@ The BloxOne provider is used to interact with the resources supported by Infoblo
 provider "bloxone" {
   csp_url = "https://csp.infoblox.com"
   api_key = "<BloxOne DDI API Key>"
+  default_tags = {
+    managed_by = "terraform"
+    location   = "us-west-1"
+  }
```

Comment:
I think we should have two blocks there. One with the basic config for csp_url and api_key. And another block that explains that default tags can be provided in the provider configuration as a way to define common tags and have this example under there.

Reply:
Instead, shall I add an `optional` comment above the default_tags section?